src/e32.h

```diff
@@ -23,7 +23,10 @@
  out quite difficult. For now we'll have to use 58 until a better solution
  can be devised.
 */
-#define E32_TX_BUF_BYTES 58
+#define E32_MAX_PACKET_LENGTH 58
+
+#define TX_BUF_BYTES 512
```

Comment:
Check this needs to be 512. All reads are `E32_TX_BUF_BYTES` in length. Also, check that every time we read we send it right off so we wouldn't mix transmissions from different sources. Although, they could be interleaved I suppose.